audit-logs/chart/templates/metrics-collector.yaml

```diff
@@ -0,0 +1,55 @@
+{{/* 
+SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+SPDX-License-Identifier: Apache-2.0
+*/}}
+
+{{- if .Values.openTelemetry.metricsCollector.enabled }}
+apiVersion: opentelemetry.io/v1beta1
+kind: OpenTelemetryCollector
+metadata:
+  name: metrics
+  labels:
+  {{- include "plugin.labels" . | nindent 4 }} 
+  {{- if .Values.openTelemetry.customLabels }}
+  {{ toYaml .Values.openTelemetry.customLabels | nindent 4 }}
+  {{- end }}
```

Comment:
The metrics collector can be removed entirely. This should be covered by a `metrics-opentelemetry` plugin if we continue to follow the naming pattern.